documentation/release_notes.markdown

```diff
@@ -61,33 +91,56 @@ This release retires all api versions before v4 and promotes v4 to be
 the stable API. The following changes have been made since the last
 stable release of PuppetDB, which included an experimental v4 API.
 
-* Retire all previously deprecated commands ([PDB-840](https://tickets.puppetlabs.com/browse/PDB-840))
+* The version 2 and 3 of the query API are now retired, and v4 experimental is now laid out quite differently.
```

Comment:
Maybe `The v4 query API is quite different from the PuppetDB 2 experimental version`